services/catalog/src/simcore_service_catalog/db/repositories/services.py

```diff
@@ -191,31 +191,30 @@ async def create_service(
                     f"{access_rights} does not correspond to service {new_service.key}:{new_service.version}"
                 )
 
-        async with self.db_engine.connect() as conn:
+        async with self.db_engine.begin() as conn:
             # NOTE: this ensure proper rollback in case of issue
-            async with conn.begin() as _transaction:
```

Comment:
Q: so here it was working (the part under begin), right?

Reply:
yes, but since the transaction is anyway created above, there is no need to have it twice. Also nested transactions are not supported I think.